runtime-modules/blog/src/benchmarking.rs

```diff
@@ -0,0 +1,392 @@
+            )
+        );
+
+        assert_last_event::<T, I>(RawEvent::ReplyEdited(
+                participant_id,
+                post_id,
+                reply_id,
+                updated_text
+            ).into());
+    }
+
+    react_to_post {
+        let post_id = generate_post::<T, I>();
+        let (account_id, participant_id) = member_funded_account::<T, I>("caller", 0);
+        let origin = RawOrigin::Signed(account_id);
+    }: react(
+        origin.clone(),
+        participant_id,
+        0,
+        post_id,
+        None
+        )
+    verify {
+        assert_last_event::<T, I>(
+            RawEvent::PostReactionsUpdated(participant_id, post_id, 0).into()
+        );
+    }
+
+    react_to_reply {
+        let post_id = generate_post::<T, I>();
+        let (account_id, participant_id) = member_funded_account::<T, I>("caller", 0);
+        let reply_id = generate_reply::<T, I>(account_id.clone(), participant_id, post_id.clone());
+        let origin = RawOrigin::Signed(account_id);
+    }: react(
+        origin.clone(),
+        participant_id,
+        0,
+        post_id,
+        Some(reply_id)
+    )
+    verify {
+        assert_last_event::<T, I>(
+            RawEvent::ReplyReactionsUpdated(participant_id, post_id, reply_id, 0).into()
+        );
+    }
+}
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use crate::mock::{ExtBuilder, Runtime};
+    use frame_support::assert_ok;
+
+    #[test]
+    fn test_create_post() {
+        ExtBuilder::default().build().execute_with(|| {
+            assert_ok!(test_benchmark_create_post::<Runtime>());
+        })
+    }
+
+    #[test]
+    fn test_lock_post() {
+        ExtBuilder::default().build().execute_with(|| {
+            assert_ok!(test_benchmark_lock_post::<Runtime>());
+        })
+    }
+
+    #[test]
+    fn test_unlock_post() {
+        ExtBuilder::default().build().execute_with(|| {
+            assert_ok!(test_benchmark_unlock_post::<Runtime>());
+        })
+    }
+
+    #[test]
+    fn test_edit_post() {
+        ExtBuilder::default().build().execute_with(|| {
+            assert_ok!(test_benchmark_edit_post::<Runtime>());
+        })
+    }
+
+    #[test]
+    fn test_create_reply_to_post() {
+        ExtBuilder::default().build().execute_with(|| {
+            assert_ok!(test_benchmark_create_reply_to_post::<Runtime>());
+        })
+    }
+
+    #[test]
+    fn test_create_reply_to_reply() {
+        ExtBuilder::default().build().execute_with(|| {
+            assert_ok!(test_benchmark_create_reply_to_reply::<Runtime>());
+        })
+    }
+
+    #[test]
+    fn test_edit_reply() {
+        ExtBuilder::default().build().execute_with(|| {
+            assert_ok!(test_benchmark_edit_reply::<Runtime>());
+        })
+    }
+
+    #[test]
+    fn test_react_to_post() {
+        ExtBuilder::default().build().execute_with(|| {
+            assert_ok!(test_benchmark_react_to_post::<Runtime>());
+        })
+    }
```

Comment:
It seems, that `react_to_reply` test is missing.

Reply:
Good catch!